docs/en/reference/basic-mapping.rst

```diff
@@ -209,27 +209,39 @@ built-in mapping types:
 -  ``decimal``: Type that maps a SQL DECIMAL to a PHP string.
 -  ``date``: Type that maps a SQL DATETIME to a PHP DateTime
    object.
+-  ``date_immutable``: Type that maps a SQL DATETIME to a PHP DateTimeImmutable
```

Comment:
In https://github.com/doctrine/dbal/pull/2793, it was agreed that it's `[ˈɛs kjuː ˈɛl]` (therefore, "an") not `[ˈsiːkwəl]` (not "a").